saleor/order/tests/test_fetch_order_prices.py

```diff
@@ -2011,263 +1935,6 @@ def test_fetch_order_prices_manual_order_discount_and_voucher_apply_once_per_ord
-    )
-    assert (
-        order.total_gross_amount
-        == (subtotal.amount + shipping_price.amount - manual_discount_amount) * tax_rate
-    )
-    assert order.subtotal_net_amount == subtotal.amount - manual_discount_amount
-    assert (
-        order.subtotal_gross_amount
-        == (subtotal.amount - manual_discount_amount) * tax_rate
-    )
-    assert order.undiscounted_total_net == subtotal + shipping_price
-    assert order.undiscounted_total_gross == (subtotal + shipping_price) * tax_rate
-    assert order.shipping_price_net == shipping_price
-    assert order.shipping_price_gross == shipping_price * tax_rate
-    assert order.base_shipping_price == shipping_price
-
-    assert (
-        discounted_line.base_unit_price_amount
-        == discounted_line.undiscounted_base_unit_price_amount
-        - manual_line_discount_value
-    )
-    assert (
-        discounted_line.total_price_net_amount
-        == discounted_line.unit_price_net_amount * discounted_line.quantity
-    )
-    assert (
-        discounted_line.total_price_gross_amount
-        == discounted_line.unit_price_net_amount * discounted_line.quantity * tax_rate
-    )
-    assert (
-        discounted_line.undiscounted_total_price_net_amount
-        == discounted_line.undiscounted_base_unit_price_amount
-        * discounted_line.quantity
-    )
-    assert (
-        discounted_line.undiscounted_total_price_gross_amount
-        == discounted_line.undiscounted_base_unit_price_amount
-        * discounted_line.quantity
-        * tax_rate
-    )
-    assert discounted_line.unit_discount_amount == manual_line_discount_value
-    assert discounted_line.unit_discount_type == DiscountValueType.FIXED
-    assert discounted_line.unit_discount_reason == manual_line_discount.reason
-
-    assert line_1.base_unit_price_amount == line_1.undiscounted_base_unit_price_amount
-    assert (
-        line_1.total_price_net_amount
-        == order.subtotal_net_amount - discounted_line.total_price_net_amount
-    )
-    assert (
-        line_1.total_price_gross_amount
-        == (order.subtotal_net_amount - discounted_line.total_price_net_amount)
-        * tax_rate
-    )
-    assert (
-        line_1.undiscounted_total_price_net_amount
-        == line_1.undiscounted_base_unit_price_amount * line_1.quantity
-    )
-    assert (
-        line_1.undiscounted_total_price_gross_amount
-        == line_1.undiscounted_base_unit_price_amount * line_1.quantity * tax_rate
-    )
-    assert line_1.unit_discount_amount == 0
-    assert line_1.unit_discount_type is None
-    assert line_1.unit_discount_reason is None
-
-    assert discounted_line.discounts.count() == 1
-
-    manual_line_discount.refresh_from_db()
-    assert manual_line_discount.amount_value == manual_discount_amount
-    assert manual_line_discount.type == DiscountType.MANUAL
-
-
-def test_fetch_order_prices_manual_line_discount_and_voucher_apply_once_per_order(
```

Comment:
This test was replaced with `test_draft_order_update_with_voucher_apply_once_per_order_and_manual_line_discount` and `test_add_manual_line_discount_order_with_voucher_apply_once_per_order`